app/src/test/java/woowacourse/shopping/CartPresenterTest.kt

```diff
@@ -39,20 +39,28 @@ class CartPresenterTest {
             }
 
             override fun find(id: Int): Product {
-                return mProducts[id]
+                return Product(0, "락토핏", "", Price(10000))
             }
 
             override fun findRange(mark: Int, rangeSize: Int): List<Product> {
-                return mProducts.subList(mark, mark + rangeSize)
+                return listOf(
+                    Product(0, "락토핏", "", Price(10000)),
+                    Product(1, "락토핏", "", Price(10000)),
+                    Product(2, "락토핏", "", Price(10000)),
+                    Product(3, "락토핏", "", Price(10000)),
+                    Product(4, "락토핏", "", Price(10000)),
+                )
             }
 
             override fun isExistByMark(mark: Int): Boolean {
-                return mProducts.find { it.id == mark } != null
+                if (mark < 0) return false
+                return true
```

Comment:
```suggestion
                return mark > -1
```
이렇게 하셔도 됩니다만, 이 부분도 로직이기 때문에 없는 것이 좋긴 합니다. 😅